new-content/events/summer-school-2021.yaml

```diff
@@ -21,7 +21,7 @@ header:
     location: 'Online'
     date: 'July 12 - 23, 2021'
     time:
-    to: 'https://qiskit.org'
+    to: 'http://qisk.it/qgss2021blog_website_source'
```

Comment:
same please

Reply:
Okay, I see that it redirects to a secure link... and I also see that our link to metal also uses `http://qisk.it/metal`

I'm good with it.